src/tests/epics/variants.test.ts

```diff
@@ -1,7 +1,6 @@
 import {
     findTestAndVariant,
     getUserCohorts,
-    Test,
```

Comment:
should this file be `epicSelection.test.ts`?

Reply:
👍